lpa-submissions-e2e-tests/cypress/support/common/subTasks.js

```diff
@@ -72,9 +72,5 @@ export const subTasks = [
 ];
 
 export const getSubTaskInfo = (subTaskRef) => {
-  const subTask = subTasks.find((st) => st.ref === subTaskRef);
-
-  if (!subTask) throw new Error(`Unknown sub task name = ${subTaskRef}`);
-
-  return subTask;
+  return subTasks.find((st) => st.ref === subTaskRef);
```

Comment:
Why are we removing error handling for this function?

Reply:
Not really removing error handling at all, just removing where Cypress would fire an error. This is just for cypress test and as we add new questions we're migrating away from this. New pattern is to see if this returns anything, otherwise try the new method. Allows us to have a page object that comes from the step file instead of this included file that was a bit of an anti pattern